scripts/build_industrial_distribution_key.py

```diff
@@ -30,19 +30,10 @@ def map_industry_to_buses(df, countries, gadm_level, shapes_path, gadm_clusterin
     Only cement not steel - proof of concept.
     Change hotmaps to more descriptive name, etc.
     """
```

Comment:
A docstring of `map_industry_to_buses( )` seems to be a bit outdated. May it be a good idea to fix it?